core-web/libs/contenttype-fields/src/lib/fields/binary-field/components/dot-binary-field-url-mode/dot-binary-field-url-mode.component.html

```diff
@@ -0,0 +1,49 @@
+<form
+    class="url-mode__form"
+    *ngIf="vm$ | async as vm"
+    [formGroup]="form"
+    (ngSubmit)="onSubmit()"
+    data-testId="form">
+    <div class="url-mode__input-container">
+        <label for="url-input">{{ 'dot.binary.field.action.import.from.url' | dm }}</label>
+        <input
+            id="url-input"
+            type="text"
+            formControlName="url"
+            pInputText
+            placeholder="https://www.dotcms.com/image.png"
+            aria-label="URL input field"
+            data-testId="url-input" />
+        <dot-field-validation-message
```

Comment:
There is a layour shift when this show or hide... we should have the space reserved in case it show to avoid this.

Reply:
Done!